study.md

```diff
@@ -31,7 +31,7 @@ In your own words, define what the responsibilities of the model layer are in
 Rails.
 
 ```md
-<!-- your answer here -->
+The model layer is the hard working number cruncher in Rails that nobody sees. The controller layer sends data to the model layer and the model layer takes the request, processes it in the back end and spits out a response to the request back at the controller.
```

Comment:
The model holds any logic and communicates with the database. Your last sentence really describes what Rails does as a whole, not what the model layer does.